src/AdminConsole/Components/App.razor

```diff
@@ -9,16 +9,28 @@
     <SecureStylesheet href="css/tailwind.css" />
     <HeadOutlet/>
     <SecureScript src="lib/passwordless/passwordless.umd.min.js" />
+    <SecureImportMapScript Value="@ImportMap" />
 </head>
 
 <body class="h-full bg-gray-50">
     <Routes/>
     <script src="_framework/blazor.web.js" autostart="false"></script>
     <SecureScript type="text/javascript" src="js/core.js" />
+    <SecureScript type="module" src="js/confirm-form.mjs" />
```

Comment:
Required for confirmation modals, although this required the Vue.js dependency, see line 12.